toolchain/flash_node_id.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+"""Flash a node id to OreSat firmware card.
+
+We store the Node IDs in the "Flash Option Byte Register" (See RM00091 Reference
+Manual - 3.5.7 Flash Option byte register (FLASH_OBR)).
+
+The node ID is consumed and passed to CANOpenNode in common/oresat.c
+"""
+
+import subprocess
+import sys
+from argparse import ArgumentParser
+
+# FIXME: retrieve IDs from oresat-configs
```

Comment:
It would be good to have this implemented rather than left as a FIXME. I personally almost never get to the "fix" part.